tests/rptest/perf/ts_write_openmessaging_test.py

```diff
@@ -35,17 +36,22 @@ def __init__(self, ctx):
                              extra_rp_conf=extra_rp_conf)
 
     @cluster(num_nodes=6)
-    @parametrize(driver_idx="ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT",
-                 workload_idx="RELEASE_CERT_SMOKE_LOAD_625k")
-    def test_perf(self, driver_idx, workload_idx):
+    @parametrize(driver_idx="ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT")
```

Comment:
Something went wrong rebasing here I assume? These seem like unrelated changes.

Reply:
Yep, should have used `--keep-base` because I can'gt count to 6.